File: Python/1213.py
from sys import stdin

def minOnes(n):
    count = 1
    ones = 11
    if n == 1:
        return count
    while ones % n != 1:
        ones = (ones % n) * 10 + 1
        count += 1
    return count

for line in stdin:
    print(minOnes(int(line)))

File: Javascript/1213.js
function minOnes(n) {
    let count = 1;
    let ones = 11;
    if(n == 1) return count;
    while(ones % n != 1) {
        ones = (ones % n) * 10 + 1;
        count += 1;
    }
    return count;
}

const readline = require("readline");
rl =  readline.createInterface(process.stdin);

rl.on("line", (num) => {
    console.log(minOnes(num));
});
